docs/.gitignore

```diff
@@ -0,0 +1,2 @@
+build
+hotdoc-private-libasync
```

Comment:
When is that used?

Reply:
it's not anymore, residue from testing. will remove